server/src/domain/repositories/search.repository.ts

```diff
@@ -92,6 +92,7 @@ export interface SearchStatusOptions {
 export interface SearchOneToOneRelationOptions {
   withExif?: boolean;
   withSmartInfo?: boolean;
+  withStacked?: boolean;
```

Comment:
We can remove the `withStacked` field in the `SearchRelationOptions` interface now that we've added it to `SearchOneToOneRelationOptions`